web/next/package.json

```diff
@@ -17,7 +17,6 @@
     "@portabletext/react": "^3.0.11",
     "@reduxjs/toolkit": "^1.7.2",
     "@sanity/client": "^3.2.0",
-    "@sanity/image-url": "^1.0.2",
```

Comment:
This has been removed since now, I'm querying for the full image data dynamically within each embedded image component. This should make it easier to get cleaner-looking, appropriately-resized images later down the road. For now, I'm just passing the raw image asset URL to display and using the `metadata.lqip` field as the blur image URL (it's a base64 encoding of a 20x20 version of the image)